examples/reference/elements/bokeh/Points.ipynb

```diff
@@ -147,10 +147,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "**Note**: Although the  ``Scatter`` element is superficially similar to the [``Points``](./Points.ipynb) element (they can generate plots that look identical), the two element types are semantically quite different. The fundamental difference is that [Scatter](./Scatter.ipynb) is used to visualize data where the *y* variable is *dependent*. This semantic difference also explains why the histogram generated by ``hist`` call above visualizes the distribution of a different dimension than it does for [``Scatter``](./Scatter.ipynb).\n",
-    "\n",
-    "This difference means that ``Points`` naturally combine elements that express independent variables in two-dimensional space, for instance [``Raster``](./Raster.ipynb) types such as [``Image``](./Image.ipynb). Similarly, ``Scatter`` expresses a dependent relationship in two-dimensions and combine naturally with ``Chart`` types such as [``Curve``](./Curve.ipynb).\n",
+    "**Note**: Although the  ``Scatter`` element is superficially similar to the [``Points``](./Points.ipynb) element (they can generate plots that look identical), the two element types are semantically quite different. The fundamental difference is that [Scatter](./Scatter.ipynb) is used to visualize data where the *y* variable is *dependent*, unlike ``Points``. This semantic difference also explains why the histogram generated by the ``hist`` call above visualizes the distribution of a different dimension than it does for [``Scatter``](./Scatter.ipynb).\n",
     "\n",
+    "This difference means that ``Points`` elements can easily be overlaid with other elements that express independent variables in two-dimensional space, such as [``Raster``](./Raster.ipynb) types like [``Image``](./Image.ipynb). Conversely, ``Scatter`` expresses a dependent relationship between *x* and *y* and thus overlays easily with ``Chart`` types such as [``Curve``](./Curve.ipynb).\n",
```

Comment:
While this is true conceptually people might be confused by this since scatter overlays on easter/image just fine.

Reply:
Yeah; maybe you can think of a better way to say that. "most naturally"?

Reply:
"Most naturally" works for me, "easily" was my main objection.

Reply:
I'll update to "most naturally.